platforms/documentation/docs/src/docs/userguide/dep-man/basics/dependency_constraints_conflicts.adoc

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[dependency-constraints-conflicts]]
+= 5. Dependency Constraints and Conflict Resolution
+
+[[sec:conflict-resolution]]
+== Understanding types of conflicts
+
+When doing dependency resolution, Gradle handles two types of conflicts:
+
+1. *Version conflicts*: That is when two or more dependencies require a given dependency but with different versions.
+2. *Capability conflicts*: That is when the dependency graph contains multiple modules that provide the same implementation.
```

Comment:
```suggestion
2. *Capability conflicts*: That is when the dependency graph contains multiple components that provide the same implementation.
```

`module` and `component` are specific keywords with precise definitions. 

A module represents the group/artifactId (com.google.guava:guava). A component is a specific _version_ of that like guava 31

Graphs do not contain modules, but _instances_ of a module (a component)